app/controllers/admin/applications_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Admin::ApplicationsController < ApplicationController
+    def show
+        @application = Application.find(params[:id])
+        
+        @pets = Pet.search(params[:search])
+        
+        if params[:added_pet].present? 
```

Comment:
This functionality of adding a pet to an application is not actually necessary to include here, as that is only needed from the user perspective (i.e. the "applications#show" page)